src/main/java/com/couchbase/client/dcp/message/SnapshotMarkerFlags.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2016 Couchbase, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.couchbase.client.dcp.message;
+
+public enum SnapshotMarkerFlags {
+    ;
+
+    /**
+     * Specifies that the snapshot contains in-memory items only.
+     */
+    public static final int MEMORY = 0x01;
```

Comment:
wait a minute, you already defined an enum why not do enum properties and instead use static final fields? :D